### PR TITLE
#NA - Setting plugin location monitor interval to 5secs for Developme…

### PR DESCRIPTION
**Pull request by jyotisingh**
…nt Server

This makes it convenient to test plugins and end-points while developing.

### CODE REVIEW

File: development-utility/development-server/src/com/thoughtworks/go/server/DevelopmentServer.java [hunk 1]
```diff
@@ -52,6 +52,7 @@ public static void main(String[] args) throws Exception {
 
         systemEnvironment.setProperty(SystemEnvironment.PARENT_LOADER_PRIORITY, "true");
         systemEnvironment.setProperty(SystemEnvironment.CRUISE_SERVER_WAR_PROPERTY, webApp.getAbsolutePath());
+        systemEnvironment.set(SystemEnvironment.PLUGIN_LOCATION_MONITOR_INTERVAL_IN_SECONDS, 5);
```
Comment (zabil) [inline development-utility/development-server/src/com/thoughtworks/go/server/DevelopmentServer.java]: What about the agent?

**Comment by arvindsv** [comment]
Also, does a plugin that needs to be sent to the agent (like the SCM or task plugins) actually get sent to the agent, when it is updated on the server? I thought it wasn't sent. Would be nice if it was.

**Comment by jyotisingh** [comment]
@arvindsv - That bug was [fixed](https://github.com/gocd/gocd/issues/825) so, download on agent should work. 
@zabil - Will update the PR.

**Comment by arvindsv** [comment]
> @arvindsv - That bug was fixed, so download on agent should work. 

Woo hoo!

**Comment by zabil** [comment]
looks good to me.

**Pull request merged**